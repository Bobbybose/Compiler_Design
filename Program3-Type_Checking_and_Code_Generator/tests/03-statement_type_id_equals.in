int a = 2;
float b = 2.3;
char c = 'a';
auto d = 2;
b = d;