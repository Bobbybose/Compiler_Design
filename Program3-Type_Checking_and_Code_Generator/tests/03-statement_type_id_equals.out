Code:
%t1 = 2
a = %t1
%t2 = 2.300000
b = %t2
%t3 = 'a'
c = %t3
%t4 = 2
d = %t4
%t5 = int2float d
b = %t5

